src/App.jsx

```diff
@@ -3,26 +3,74 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getAllTodos } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
+    query: '',
+    filterOfTodos: 'all',
   };
 
+  componentDidMount() {
+    (async() => {
```

Comment:
```suggestion
  componentDidMount = async() => {
```

Так виглядає трохи краще